astro.config.mjs

```diff
@@ -27,6 +29,16 @@ export default defineConfig({
         }
       }],
     ]
-  })],
+  }), icon()],
   site: 'https://ubie-vitals.github.io',
-});
\ No newline at end of file
+  vite: {
+    ssr: {
+      noExternal: ['@ubie/ubie-icons', '@ubie/ubie-ui']
+    }
+  },
+  markdown: {
+    shikiConfig: {
+      transformers: [transformerNotationWordHighlight(), transformerNotationHighlight(), transformerNotationDiff()]
```

Comment:
Option to decorate code for clarity（for astro）

https://shiki.style/packages/transformers